src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -147,29 +147,35 @@
               </a>
             </div>
             <div class="d-flex flex-column ms-4 me-4 mt-4 mb-4">
-              <ng-template #healthChecks>
-                <cd-health-checks *ngIf="healthData?.health?.checks"
-                                  [healthData]="healthData.health.checks"></cd-health-checks>
-              </ng-template>
 
               <div class="d-flex flex-row col-md-3 ms-4">
                 <i *ngIf="healthData.health?.status"
-                   [ngClass]="[healthData.health.status | healthIcon, icons.large2x]"
-                   [ngStyle]="healthData.health.status | healthColor"
-                   [title]="healthData.health.status"></i>
+                    [ngClass]="[healthData.health.status | healthIcon, icons.large2x]"
+                    [ngStyle]="healthData.health.status | healthColor"
+                    [title]="healthData.health.status">
+              </i>
+              <span class="ms-2 mt-n1 lead"
+              *ngIf="!healthData.health?.checks?.length"
+              i18n>Cluster</span>
+              <cds-toggletip
+                  [dropShadow]="true"
+                  [align]="bottom">
```

Comment:
```suggestion
                  align="bottom">
```
It string only so why [] ?